apps/shade/src/components/ui/popover.stories.tsx

```
@@ -0,0 +1,34 @@
+    title: 'Components / Popover',
+    component: Popover,
+    tags: ['autodocs']
+} satisfies Meta<typeof Popover>;
+
+export default meta;
+type Story = StoryObj<typeof Popover>;
+
+export const Default: Story = {
+    args: {
+        children: [
+            <div style={{height: 400}}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)